docs/pages/docs/schema.mdx

```diff
@@ -46,6 +46,7 @@ Ponder also includes a few extra column types built specifically for EVM indexin
 | :------------ | :------------------------------------------- | :-------------- | :-------------------- |
 | `bigint{:ts}` | Large integer (holds `uint256` and `int256`) | `bigint`        | `NUMERIC(78,0){:sql}` |
 | `hex{:ts}`    | UTF‐8 character sequence with `0x` prefix    | `0x${string}`   | `TEXT{:sql}`          |
+| `bytes{:ts}`  | Low-level byte array                         | `Uint8Array`    | `bytea{:sql}`         |
```

Comment:
We have a quick section below for our other custom column types (`bigint` and `hex`), would be nice to add one for `bytes`

Also I think the description could just be "Byte array"